docs/inter-domain-integration/metering-point-period-schema.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyspark.sql.types import (
+    StructField,
+    StringType,
+    TimestampType,
+    StructType,
+)
+
+"""
+Schema for metering point periods
+
+Metering point periods are used in balance fixing and settlement. Some fields are only used in settlement. See each field for details.
+Periods (given by `FromDate` and `ToDate`) must not overlap and must not have gaps in between.
+All but the `ToDate` of the last period must have value. The `ToDate` of the last period is null.
+
+Data must be stored in a Delta table.
+The table holds all consumption, production, exchange, and child metering points.
+The table must be partitioned by `ToDate`: ToDate_Year/ToDate_Month/ToDate_Date.
+TODO explain why by to-date
+The table data must always contain updated periods.
+"""
+metering_point_period_schema = StructType(
+    [
+        # GSRN (18 characters) that uniquely identifies the metering point
+        # Used in balance fixing and settlement.
+        # Example: 578710000000000103
+        StructField("MeteringPointId", StringType(), False),
+        
+        # "E17" (consumption) | "E18" (production) | "E20" (exchange) | "D01"-"D99" (child)
+        # Used in balance fixing and settlement. However, child metering points are only used in settlement.
+        # Example: E20
+        StructField("Type", StringType(), False),
+        
+        # "E02" (quarterly)| "D01" (flex)
+        # Used in balance fixing and settlement.
+        # Example: D01
+        StructField("SettlementMethod", StringType(), False),
```

Comment:
Will be null when MP != consumption
```suggestion
        StructField("SettlementMethod", StringType(), True),
```